tests/kotlin-js-store-1.8/yarn.lock

```diff
@@ -2,31 +2,312 @@
 # yarn lockfile v1
 
 
+"@colors/colors@1.5.0":
```

Comment:
Added by mistake I think :)

Reply:
The debug printf we mistakes but I think this one is on purpose. I added a new test framework (karma, for browser tests). I think it requires those dependencies. Let's see what CI has to say about it.

Reply:
Oh sorry I thought this came from testing the 1.8.20 branch. Nevermind!